internal/services/appservice/linux_function_app_resource_test.go

```diff
@@ -4244,6 +4247,7 @@ resource "azurerm_linux_function_app" "test" {
   storage_account_name       = azurerm_storage_account.test.name
   storage_account_access_key = azurerm_storage_account.test.primary_access_key
 
+  // vnet_image_pull_enabled = true 
```

Comment:
When we hook the feature flag `FourPointOhBeta` up to an environment variable to be able to enable the beta and to test that the 4.0 features work, this will be problematic and require manual intervention to uncomment the line in order to test. This needs to be removed and added to a new test that is skipped prior to 4.0 e.g. 
```
	if !features.FourPointOhBeta() {
		t.Skip("this test requires 4.0 mode")
	}
```
Please make this change to the other resource tests too